### PR TITLE
Add Duong-2020-100

### PR DESCRIPTION
**Pull request by Kristina-Pianykh**
# Pull request checklist

- [x] add new concept list
- [ ] add new metadata
- [ ] add new Concepticon concept sets
- [ ] add new Concepticon concept relations
- [ ] refine existing Concepticon concept set mappings
- [ ] refine Concepticon glosses
- [ ] refine Concepticon concept relations
- [ ] refine Concepticon concept definitions
- [ ] retire data

## Additional information
Refers to #869. A very basic 100-item list.
P.S.: I checked the Lexibank repository, there is already an [issue ](https://github.com/lexibank/lexibank/issues/216)there and the data in txt. But I'm not sure if it is the right data "format", i.e. if the numbers represented as superscripts in the publication, should be depicted in the raw file as such, too. If so, I'll type it off and dump it in the Lexibank issue.

### CODE REVIEW

**Comment by Kristina-Pianykh** [comment]
Since there is already one open PR still under discussion, could you moderate/have a quick look at this one, @chrzyki?

**Comment by chrzyki** [comment]
Sure, thanks for adding the list! @Schweikhard, would you have time for a review?

**Comment by LinguList** [comment]
@Kristina-Pianykh, the numbers are no problem, as they can be converted by an orthography profile when adding the list. In fact, this lexibank dataset can be added in no time, and I will see if I find time to write the first script later.

File: concepticondata/conceptlists.tsv [hunk 1]
```diff
@@ -355,4 +355,5 @@ List-2020-1365	List, Johann-Mattis	2020		1365	annotated	English	German	https://c
...
+Duong-2020-100	Thu Hang, Duong and Thu Quynh, Nguyen and Van Loi, Nguyen	2020		100	basic, questionnaire	English	La Chí languages in Vietnam	https://evols.library.manoa.hawaii.edu/bitstream/10524/52466/1/JSEALS_Special_Publication_6_Anthropologyoflanguage.pdf	Duong2020		The current list was used to investigate the phonetic, lexical, and grammatical characteristics of the La Chí language in Bản Díu commune, in comparison with the language of La Chí in ản Máy and Bản Phùng communes in Vietnam. The concepts are claimed to be generally based on the 100-item list of Swadesh (presumably, the [one that dates back to 1955](:ref:Swadesh-1955-100)). The data for the analysis come from the materials, collected by Nguyễn Văn Lợi, Nguyễn Thu Quỳnh and Dương Thu Hằng during their field trips in the respective regions.	136-138
```
Comment (LinguList) [inline concepticondata/conceptlists.tsv:359]: I propose to slightly modify the text: 

> The current list was use to investigate some basic characteristics of the La Chí language spoken in Bản Díu, comparing it with La Chí varieties in  Bản Máy and Bản Phùng

This is less similar to the author's text, and it also does not follow the omission of articles (as we find it in the original).

Comment (LinguList) [reply concepticondata/conceptlists.tsv:359]: > The data for the analysis come from the materials collected by Nguyễn Văn Lợi, Nguyễn Thu Quỳnh and Dương Thu Hằng during their field trips in the respective regions.

[ no comma here ]

**Review by LinguList — APPROVED** [review]
concept mappings all look fine to me!

File: concepticondata/conceptlists/Duong-2020-100.tsv [hunk 2]
```diff
@@ -0,0 +1,101 @@
+ID	NUMBER	ENGLISH	CONCEPTICON_GLOSS	CONCEPTICON_ID
+Duong-2020-100-1	1	sun	SUN	1343
+Duong-2020-100-2	2	moon	MOON	1313
+Duong-2020-100-3	3	star	STAR	1430
+Duong-2020-100-4	4	earth	EARTH (SOIL)	1228
+Duong-2020-100-5	5	cloud	CLOUD	1489
+Duong-2020-100-6	6	rain	RAINING OR RAIN	2108
```
Comment (Schweikhard) [inline concepticondata/conceptlists/Duong-2020-100.tsv:7]: The list seems to be sorted by parts of speech, so we can be confident that this is RAIN (PRECIPITATION).

File: concepticondata/conceptlists/Duong-2020-100.tsv [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+Duong-2020-100-7	7	water	WATER	948
+Duong-2020-100-8	8	mountain	MOUNTAIN	639
+Duong-2020-100-9	9	rock	ROCK	668
+Duong-2020-100-10	10	sand	SAND	671
+Duong-2020-100-11	11	road/way	ROAD	667
```
Comment (Schweikhard) [inline concepticondata/conceptlists/Duong-2020-100.tsv:12]: Perhaps rather PATH OR ROAD?

File: concepticondata/conceptlists/Duong-2020-100.tsv [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+Duong-2020-100-12	12	fire	FIRE	221
+Duong-2020-100-13	13	smoke	SMOKE (EXHAUST)	778
+Duong-2020-100-14	14	ash	ASH	646
+Duong-2020-100-15	15	night	NIGHT	1233
+Duong-2020-100-16	16	tree	TREE	906
+Duong-2020-100-17	17	seed	SEED	714
+Duong-2020-100-18	18	leaf	LEAF	628
+Duong-2020-100-19	19	root	ROOT	670
+Duong-2020-100-20	20	bark	BARK	1204
```
Comment (Schweikhard) [inline concepticondata/conceptlists/Duong-2020-100.tsv:21]: Now that's an unfortunate ordering to have tree bark followed by dog. Let's hope it didn't confuse the elicitors.

**Review by Schweikhard — APPROVED** [review]
Looks good, just some small notes.

File: concepticondata/conceptlists/Duong-2020-100.tsv (hunk 4 above)
Comment (LinguList) [reply concepticondata/conceptlists/Duong-2020-100.tsv:21]: :-) yes! I think we'll see from the lexical data, as they tend to use a compound with tree for tree bark.

**Review by chrzyki — APPROVED** [review]

File: concepticondata/conceptlists.tsv (hunk 1 above)
Comment (Kristina-Pianykh) [reply concepticondata/conceptlists.tsv:359]: Achh, my passion for commas  :) Thanks for the tips!

File: concepticondata/conceptlists/Duong-2020-100.tsv (hunk 2 above)
Comment (Kristina-Pianykh) [reply concepticondata/conceptlists/Duong-2020-100.tsv:7]: I wasn't sure since there are no explicit indications, otherwise, but I can see your point.

**Pull request merged**